packages/opal-common/opal_common/authentication/authenticator.py

```diff
@@ -0,0 +1,62 @@
+from abc import abstractmethod
+from typing import Optional
+
+from fastapi import Header
+from opal_common.authentication.deps import JWTAuthenticator
+from opal_common.authentication.types import JWTClaims
+from opal_common.authentication.verifier import JWTVerifier, Unauthorized
+from opal_common.config import opal_common_config
+from opal_common.logger import logger
+
+from .oauth2 import CachedOAuth2Authenticator, OAuth2ClientCredentialsAuthenticator
+
+
+class Authenticator:
+    @property
+    def enabled(self):
```

Comment:
Methods in this class should have a specified return type, also their arguments should be typed

Reply:
Fixed.